src/core/qgsmaprenderer.h

```diff
@@ -335,6 +335,14 @@ class CORE_EXPORT QgsMapRenderer : public QObject
      */
     bool splitLayersExtent( QgsMapLayer* layer, QgsRectangle& extent, QgsRectangle& r2 );
 
+    /** Set a feature filter provider to filter the features
+     * @param ffp the feature filter provider
+     * @note not available in Python bindings
```

Comment:
I think if it can be added to the bindings, it should be... the only exception would be something which poses a security risk (eg, the auth system).